onnxruntime/test/testdata/onnx_backend_test_series_filters.jsonc

```diff
@@ -118,7 +118,6 @@
         "^test_div_uint8_cuda",
         "^test_add_uint8_cuda",
         "^test_roialign_aligned_*",
-        "^test_bitwise_*",
```

Comment:
Please update \csharp\test\Microsoft.ML.OnnxRuntime.Tests.NetCoreApp\InferenceTest.netcore.cs as well to remove the exclusions.